### ISSUE TITLE
no pre-training on Howto100M dataset？

### ISSUE THREAD

**Issue by IvyGongoogle** [jayleicn/ClipBERT#15]
hello, I have some questions.
1.When you do 'Text-to-Video Retrieval' experiment on MSRVTT 1K test set, you do not pre-training the model end-to-end on `Howto100M` dataset. It is because the computation cost limit? And Can you provide the experiments later when you have enough computation resource?

2. Because of the computation cost limit, so you do not directly make 'Text-to-Video Retrieval' experiment on `Howto100M` test dataset ?

**Comment by jayleicn** [comment]
Hi @IvyGongoogle, 

1. Yes, we did not pre-train our model on `HowTo100M` due to the huge computation cost, and it is still not possible for us to do so.
2. There is no standard "Text-to-video retrieval" task defined for the `HowTo100M` dataset. Please let me know if I am wrong about this. Thanks! 

Best,
Jie

**Comment by IvyGongoogle** [comment]
@jayleicn 2. Sorry, my mistake. Thank you.
